src/test/kotlin/no/nav/familie/ef/sak/cucumber/steps/StepDefinitions.kt

```diff
@@ -72,6 +79,8 @@ class StepDefinitions {
     private val slot = slot<TilkjentYtelse>()
     private var stønadstype: StønadType = StønadType.OVERGANGSSTØNAD
     private val behandlingIdsToAktivitetArbeid = mutableMapOf<UUID, SvarId?>()
+    private lateinit var tilkjentYtelser: MutableMap<UUID, TilkjentYtelse>
+    private lateinit var lagredeVedtak: MutableList<Vedtak>
```

Comment:
Burde man navngi disse litt mer beskrivende? 
Nå har vi `private var vedtak = listOf<Vedtak>()` og `private lateinit var lagredeVedtak: MutableList<Vedtak>`

Hva med å kalle `vedtak` for kanskje `gittVedtak` eller noe slik

Reply:
Ja, det ene er jo state i prosessen, det andre er egentlig en database-mock-state. 

Kunne man spilt på det? Hva kan være bra navngivning da?